### PR TITLE
Updates Lara's title

### PR DESCRIPTION
**Pull request by dalrayes**
@jarmstrng

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1934,13 +1934,13 @@ <h2 class="section-title">Speakers</h2>
...
 					<h4>Lara Hogan</h4>
-					<h5>Engineering Director, Etsy</h5>
+					<h5>VP Engineering, Kickstarter</h5>
 					<ul class="spkr-social clearfix">
 						<li><a href="https://twitter.com/lara_hogan" target="_blank" rel="noopener" class="tips" title="TWITTER"><i class="icon-twitter icon"></i></a></li>
 					</ul>
 				</div>
 				<div class="col-xs-12 col-sm-9 col-lg-8">
-					<p>Lara Hogan is an Engineering Director at Etsy and the author of Designing for Performance, Building a Device Lab, and Demystifying Public Speaking. She champions performance as a part of the overall user experience, helps people get comfortable giving presentations, and believes it's important to celebrate career achievements with donuts. </p>
+					<p>Lara Hogan is VP of Engineering at Etsy and the author of Designing for Performance, Building a Device Lab, and Demystifying Public Speaking. She champions performance as a part of the overall user experience, helps people get comfortable giving presentations, and believes it's important to celebrate career achievements with donuts. </p>
```
Comment (jarmstrng) [inline index.html]: s/Etsy/Kickstarter 😁

**Review by jarmstrng — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (dalrayes) [reply index.html]: whoops! on it

Comment (jarmstrng) [reply index.html]: after that fix, 👍 :shipit: 🚀 💯

Comment (dalrayes) [reply index.html]: cool thanks :)

**Pull request merged**